cypress/e2e/specialFeatures/liteSiteWeight/assertions/liteSiteWeight.js

```diff
@@ -25,28 +32,36 @@ export default ({ path, pageType }) => {
     });
 
     it(`Page weight for ${pageType} page should be less than ${MAX_PAGE_WEIGHT_KB}Kb`, () => {
-      getTotalPageSize(allRequests).then(localPageWeight => {
-        interceptGetRequests(liveRequests);
-        cy.visit(`https://www.bbc.com${path}`);
+      getTotalPageSize(allRequests).then(
+        ({ totalSize: localPageWeight, requestSizes: localRequestSizes }) => {
+          interceptGetRequests(liveRequests);
+          cy.visit(`https://www.bbc.com${path}`);
 
-        getTotalPageSize(liveRequests).then(livePageWeight => {
-          const percentageDifference =
-            (100 * (localPageWeight - livePageWeight)) /
-            ((localPageWeight + livePageWeight) / 2);
+          getTotalPageSize(liveRequests).then(
+            ({ totalSize: livePageWeight, requestSizes: liveRequestSizes }) => {
+              const percentageDifference =
+                (100 * (localPageWeight - livePageWeight)) /
+                ((localPageWeight + livePageWeight) / 2);
 
-          const delta = roundTo2Decimals(percentageDifference);
-          expect(localPageWeight).to.be.lessThan(MAX_PAGE_WEIGHT_KB);
-          cy.task('table', [
-            {
-              URL: `${Cypress.config().baseUrl}${path}`,
-              'Page Type': pageType,
-              'Local Page Weight (KB)': localPageWeight,
-              'Live Page Weight (KB)': livePageWeight,
-              'Delta (%) ': delta,
+              const delta = roundTo2Decimals(percentageDifference);
+              expect(localPageWeight).to.be.lessThan(MAX_PAGE_WEIGHT_KB);
+              const localRequestSizesData = formatTableData(localRequestSizes);
+              const liveRequestSizesData = formatTableData(liveRequestSizes);
```

Comment:
I am wondering whether the live request stuff is useful or not... I really only wanted to see how the local page weight was derived - so that if we do add chartbeat, then we can see how big it is. But maybe it's OK? There's just a lot of info being output to the terminal now. Shall we see what others think?

Reply:
Actually maybe it is good to see the diff between local + live for comparison. Ignore me! Let's keep it 😸

Reply:
Glad you pointed this out ! I wasn't sure how helpful it was either but think we've had the same conclusion, agree it's a bit busy though